curriculum/js1/2.js

```diff
@@ -7,7 +7,9 @@
  */
 
 const solution = (a, b, c) => {
-  return 0;
+  if (a > b && a > c) return a;
+  if (b > a && b > c) return b;
+  if (c > a && c > a) return c;
```

Comment:
you don't need to check for `c` case, because if its not a or b, it has to be c!

Reply:
Resolved!

Reply:
Use Math.max